manifests/group.pp

```diff
@@ -20,6 +21,7 @@
 define yum::group (
   Enum['present', 'installed', 'absent', 'purged'] $ensure  = 'present',
   Optional[Integer]                                $timeout = undef,
+  Optional[Array[String]]                          $install_options = [],
```

Comment:
Does this need to be `Optional` if the default value is `[]` and there's not logic to check if it's `Undef`?